CromIAM/src/main/resources/application.conf

```diff
@@ -4,14 +4,14 @@ cromiam {
 }
 
 sam {
-  interface = "0.0.0.0"
+  interface = "sam.dsde-dev.broadinstitute.org"
   port = 443
   scheme = "https"
-  check-submit-whitelist: true
+  check-submit-whitelist: false
 }
 
 cromwell {
-  interface = "0.0.0.0"
+  interface = "127.0.0.1"
   port = 8000
   scheme = "http"
 }
```

Comment:
I'm not 100% sure I want to commit these changes. It does seem strange for the repo to assume a Sam running locally, which we basically never do.